platform-jobs/samza/qrcode-image-generator/src/main/java/org/ekstep/jobs/samza/model/QRCodeGenerationRequest.java

```diff
@@ -17,6 +17,11 @@
     private List<String> fileName;
     private String fileFormat;
     private int qrCodeMarginBottom;
+    private int imageMargin;
+
+    public int getImageMargin() { return imageMargin; }
```

Comment:
Can config name (qr_image_margin) and var name (imageMargin) be aligned?

Reply:
qr is prefixed to indicate the configuration is related image generation logic, among the host of the other configurations present in the properties file